index.html

```diff
@@ -29,14 +30,23 @@
             </tr>
             <!--            second person -->
             <tr>
-                <td>Gabriel Kayode</td>
-                <td>Asa</td>
-                <td>Oyo</td>
-                <td>5</td>
+                <td>Akintunde Alabi</td>
```

Comment:
Hi @tundealabi you are so close. Just one more thing - you need to keep your repo updated with the remote branch by running `git fetch` - if you don't do that, you might be running the risk of deleting someone else's work.

Like the image below shows, your contribution has erased Gabriels'. Try one more time.

<img width="1279" alt="Screen Shot 2020-05-20 at 7 12 59 PM" src="https://user-images.githubusercontent.com/7871367/82482105-3f0d5400-9ace-11ea-8c00-44e30695fe87.png">